cmd/acb/build.go

```diff
@@ -82,9 +82,11 @@ func newBuildCmd(out io.Writer) *cobra.Command {
 	f.StringArrayVar(&r.secretBuildArgs, "secret-build-arg", []string{}, "set secret build arguments")
 	f.StringArrayVar(&r.labels, "label", []string{}, "set metadata for an image")
 
+	// `username` and `password` are deprecated. Use --credentials="registryName;username;password" instead
 	f.StringVarP(&r.registryUser, "username", "u", "", "the username to use when logging into the registry")
```

Comment:
just go ahead and delete them so we don't hit similar bugs in the future; it's dead code